docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>This is more of an aesthetic suggestion, consider using different colors for the line charts.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>It might actually be useful to align "Mary" with the start of "Maryanne" and "Anne" with the end of "Maryanne", etc so it is easier for people to visually see that they are similar</p>

Reply:
They are aligned now but this required some extra code for writing offset indexes